src/adapters/postgres/cohortMembers-adapter.ts

```
@@ -6086,4 +6086,4 @@ export class PostgresCohortMembersService {
       return { condition: '', params: [] };
     }
   }
-}
+}
```

ESLint

[error] Insert `⏎`

(prettier/prettier)